### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by chmeliik**
Mostly discovered while trying to get OSBS-Box to work on a VM.

### CODE REVIEW

File: cleanup.yaml [hunk 1]
```diff
@@ -5,20 +5,16 @@
 
   tasks:
     - name: Check if openshift cluster is running
-      command: oc whoami
-      register: whoami
+      command: oc cluster status
```
Comment (twaugh) [inline cleanup.yaml]: Is `oc status` or `oc projects` or something better? `oc cluster status` is specifically for a locally running cluster, not the remote cluster you might be logged into.

Comment (chmeliik) [reply cleanup.yaml]: Currently, the playbooks only support `localhost`, but in the near future, they should be usable for remote deployments as well. When that is the case, this command will be executed on the machine which is running the cluster.

Comment (MartinBasti) [reply cleanup.yaml]: Think about CRC and OCP4, to not do something you will need to change again in future

Comment (chmeliik) [reply cleanup.yaml]: Ok, no longer checking cluster status, just attempt to log in directly

**Pull request merged**